compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -136,6 +144,242 @@ emitLinkedTuningSpec(ModuleOp module, ArrayRef<NamedSequenceOp> specsToLink) {
   return newSpec;
 }
 
+static FailureOr<NamedSequenceOp> emitLinkedDefaultTuningSpec(ModuleOp module) {
+  OpBuilder builder(module.getContext());
+  SmallVector<transform::NamedSequenceOp> namedSequenceOpsToMove;
+  SmallVector<transform::ForeachMatchOp> foreachMatchOps;
+  // foreachMatchMap: NamedSequenceOp -> ForeachMatchOps that reference it
+  // (either as a matcher or an action). It ensures
```

Comment:
Can we call the variable something like `namedSequenceToForeach` so that we don't need this comment?